cli/dcoscli/data/help/node.txt

```diff
@@ -48,6 +51,9 @@ Options:
     --filter=<filter>
         Filter logs by field and value. Filter must be a string separated by colon.
         For example: --filter _PID:0 --filter _UID:1.
+        OR
+        Filter metrics by field. Filter must be a valid metrics field.
+        For example: --filter memory.total --filter memory.swap
```

Comment:
Yes, but it's really long (113 at present). Is that OK? Can we bury at a deeper level of the docs somehow?